### PR TITLE
test: finished cypress tests for info page - incl fixes

### PR DESCRIPTION
**Pull request by kkibbey1996**
## Issue

https://gravitee.atlassian.net/browse/APIM-2075 

## Description

Info page tests for APIs in APIM. 

Splitting out my General and Plan commits, with updated code and faker details. 

This is final PR (finished test). Including conflict resolve and lint fixes. 

Tests for Danger Zone to be continued at a later date when I understand full consequence and can fulfil end-to-end test. 

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-hqxzhmqczs.chromatic.com)

### CODE REVIEW

**Comment by kkibbey1996** [comment]
My tests are passing (ui-api-info.spec.ts) so imo this is ready to be merged.

File: gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+describe('API General Page functionality', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit('/#!/environments/default/apis/');
+    cy.url().should('include', '/apis/');
+  });
+
+  before(() => {
+    api = [];
```
Comment (ctschacher) [inline gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts]: No need to use an array if you just store a single value

File: gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  });
+
+  const ApiDetails = new apiDetails();
+  const apiName = faker.commerce.productName();
+  const newName = apiName.replace(/ /g, '-');
+  const apiVersion = faker.datatype.number();
+  const apiDescription = faker.commerce.productDescription();
+  const path = require('path');
+  let api: ApiImport[];
+  const noOfApis = 1;
```
Comment (ctschacher) [inline gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts:50]: `noOfApis` is not used anywhere

**Review by ctschacher — COMMENTED** [review]
General issues:
- there are too many commits --> squash them to a single commit
- clean-up is missing (delete resources after test, here: created API)

**Pull request closed without merging**